apps/back/.prettierrc

```
@@ -0,0 +1,4 @@
+{
+  "singleQuote": true,
+  "trailingComma": "all"
+}
```

```suggestion
}

```
Please add the missing trailing line.